Formula/loc.rb

```diff
@@ -0,0 +1,17 @@
+class Loc < Formula
+  desc "Count lines of code quickly"
+  homepage "https://github.com/cgag/loc"
+  url "https://github.com/cgag/loc/archive/v0.3.2.tar.gz"
+  sha256 "0b805d53326f269e8fe21f709dc69947820fda1f291040e9225f93aef614daea"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
+    bin.install "target/release/loc"
+  end
+
+  test do
+    system bin/"loc", "--help"
```

Comment:
Please write a test that does something more substantial than printing the help. You may want to create a dummy file and count its lines of code.